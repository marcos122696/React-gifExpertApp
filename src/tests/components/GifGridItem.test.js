import React from 'react';
import { shallow } from 'enzyme';
import { GifGridItem } from '../../components/GifGridItem';


describe('Pruebas en el componenete <GifGridItem />', () => {
    
    test('Debe de mostrar <GifGridItem /> correctamente', () => {

        const wrapper = shallow(<GifGridItem />);
        expect(wrapper).toMatchSnapshot();
    })
}) 
