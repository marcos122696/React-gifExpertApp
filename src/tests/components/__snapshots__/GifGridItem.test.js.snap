// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debe de mostrar <GifGridItem /> correctamente 1`] = `
<div
  className="card animate__animated animate__zoomInUp"
>
  <p />
  <img />
</div>
`;

exports[`Debe de mostrar <GifGridItem /> correctametne 1`] = `
<div
  className="card animate__animated animate__zoomInUp"
>
  <p />
  <img />
</div>
`;

exports[`Pruebas en el componenete <GifGridItem /> Debe de mostrar <GifGridItem /> correctamente 1`] = `
<div
  className="card animate__animated animate__zoomInUp"
>
  <p />
  <img />
</div>
`;
